app/assets/stylesheets/layouts/monochrome.scss

```diff
@@ -1,23 +1,111 @@
-body { background-color: #E0E0E0; }
+@import 'variables';
+
+/* - header */
+//$font_color_header_link:  // header links
+//$font_color_header_link_hover:  // header links (hover)
+
+//$bg_color_header_top: transparent; // top header strip for logo
+//$bg_color_header_middle: #05050F; // middle header strip with user items
+//$bg_color_header_bottom: #484357; // bottom header strip with navigation
+
 #header #logo { background-color: #757575; }
 #header #user-info, #header #nav-top { background-color: #4A4A4A; }
-.flash.success, .flash.inbox { background-color: #ECECEC; }
-a { color: #6F6F6F; }
-a:visited { color: #484848; }
-.pagination .current { background: #616161; }
-.pagination a:visited { color: #3A3A3A; }
-.post-expander { background-color: #A0A0A0; }
-.post-icon, #current-icon-dropdown { background-color: #BDBDBD; }
-.post-navheader { background-color: #BDBDBD; }
-#reply-icon-selector, .replace-icon, .icons-box, .icon-credit { background-color: #BDBDBD; }
-.link-box.action-new { background-color: #5A5A5A; }
-.link-box.action-edit { background-color: #545A79; }
-.link-box.action-delete { background-color: #503636; }
-.link-box.action-favorite { background-color: #545A79;}
-.link-box.action-dismiss { background-color: #503636; }
-.subber, .flash.breadcrumbs, .form-table-ender { background-color: #ADADAD; }
-.post-character { background-color: #929292; }
-.post-screenname { background-color: #A5A5A5; }
-.post-author { background-color: #BBBBBB; }
-.sub { background-color: #BDBDBD; }
-.tag-item { color: #575757; }
+
+/* - links */
+$font_color_link: #6F6F6F; // link
+$font_color_link_visited: #484848;
+//$font_color_link_hover: #FCFEFD; // link (hovered)
+
+a { color: $font_color_link; }
+a:visited { color: $font_color_link_visited; }
+
+/* - flashes */
+// these are the notifications that appear when you submit a form or get a message
+//$bg_color_flash_error: #484357; // error
+$bg_color_flash_success: #ECECEC; // success
+$bg_color_flash_inbox: #ECECEC; // new message // TODO: THESE SHOULD NOT ALL THE SAME
```

Comment:
`.flash.error` isn't the same and it's not like people are gonna be able to distinguish different shades of grey without having another as a reference, I think? so it's not very easy to do. I guess we could scale back the monochrome focus, idk if anyone uses it though.

Reply:
it's *less* annoying with monochrome because it's monochrome, yeah